lrlex/src/lib/parser.rs

```diff
@@ -1906,6 +1916,22 @@ b "A"
             3,
             3,
         );
+// The following is missing comma separators for more than 2 elements
+// This is to avoid parsing it as "key value" number of elements.
+// However we actually error after the first element since the parser
+// knows "case_insensitive" is a flag with no arguments.
+        let src = r#"
+%grmtools {unicode, case_insensitive posix_escapes allow_comments}
+%%
```

Comment:
This comment indicates a difference between the `grmtools_section.y` parser, and the
lrlex/src/lib/parser.rs.

the `grmtools_section.y` parser returns a `case_insensitive posix_escapes` key/value pair, then fails on `allow_comments`.

I suppose we should consider using `key: value` here

Reply:
I guess another option would be `%yacckind`...

Reply:
I tried `key: value` in [10e6d74](https://github.com/softdevteam/grmtools/pull/542/commits/10e6d74b2f435bc203ccbb0f86e0e36ce65854ac)